enhancements/installer/aws-cross-account-dns-zone.md

```diff
@@ -218,6 +238,28 @@ An example IAM Trust Policy:
 
 ```
 
+#### External DNS Operator Details
+
+Prior to this enhancement, the [External DNS Operator API](https://github.com/openshift/external-dns-operator/blob/main/api)
+did not allow users to configure an AWS role ARN field. As a result, there wasn't a supported way to use the External
+DNS Operator to create DNS records in another AWS account within a shared VPC.
+
+The External DNS Operator configures and runs the [External DNS](https://github.com/openshift/external-dns) binary.
+Conveniently, External DNS already supports the [`--aws-assume-role`](https://github.com/openshift/external-dns/blob/fe00b4b83c2263282a9068655e8e3fbbc167b653/docs/faq.md#can-external-dns-manageaddremove-records-in-a-hosted-zone-which-is-setup-in-different-aws-account)
+argument, which uses the specified AWS role ARN when creating new DNS records. Therefore, to support shared VPCs with
+the External DNS Operator, we update the [API](#API-External-DNS-Operator) to expose the `--aws-assume-role` argument
+for the External DNS binary.
+
+By design, External DNS can update both public and private DNS zones using credentials that the cluster-admin provides.
+Therefore, External DNS Operator users are able to use the new [API](#API-External-DNS-Operator) to update both public
+and private DNS zones in other accounts, provided the role ARN has appropriate permissions.
+
+More details on using a role ARN to create DNS records in another AWS account will be documented in External DNS
+Operator's [Docs](https://github.com/openshift/external-dns-operator/tree/main/docs).
+
+Note that External DNS Operator updates are asynchronous to OpenShift's standard release of OpenShift Container
+Platform. Therefore, any updates to External DNS Operator are considered out of payload.
```

Comment:
Consider mentioning this in Risks and Mitigations or Version Skew, or both.

Reply:
Good idea, I added it to Version Skew, as that seemed appropriate. I'm not sure if it's a risk or needs a mitigation, I documented that it isn't a risk in Version Skew.